modules/collator-selection/src/lib.rs

```diff
@@ -143,14 +146,12 @@ pub mod pallet {
 
 	/// Basic information about a collation candidate.
 	#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, MaxEncodedLen)]
-	pub struct CandidateInfo<AccountId, Balance> {
+	pub struct CandidateInfo<AccountId> {
```

Comment:
this will cause type conflicts on acala.js, I think the easiest way is remove this and just store `OrderedSet<AccountId>` instead.